.github/workflows/workflows.yaml

```diff
@@ -0,0 +1,40 @@
+name: Keycloak FranceConnect CI
+
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Set current version
+        run: mvn -B versions:set -DnewVersion=${GITHUB_REF##*/}
```

Comment:
Why do you change the version here ?

Reply:
Indeed it's useless, with this job we only want to see if maven build work. I remove it.